cinema/serializers.py

```diff
@@ -31,27 +31,67 @@ class Meta:
 
 
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
```

Comment:
image shouldn't be included here - it needs to be only on list and detail serializers